src/UI/Seller/src/app/user-group-management/containers/user-group-details/user-group-details.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { faUsers, faUser } from '@fortawesome/free-solid-svg-icons';
+import {
+  UserGroup,
+  OcUserGroupService,
+  UserGroupAssignment,
+} from '@ordercloud/angular-sdk';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { flatMap, tap } from 'rxjs/operators';
+import {
+  AppConfig,
+  applicationConfiguration,
+} from '@app-seller/config/app.config';
+
+@Component({
+  selector: 'user-group-details',
+  templateUrl: './user-group-details.component.html',
+  styleUrls: ['./user-group-details.component.scss'],
+})
+export class UserGroupDetailsComponent implements OnInit {
+  usergroup: UserGroup;
+  faUsers = faUsers;
+  faUser = faUser;
+
+  constructor(
+    private activatedRoute: ActivatedRoute,
+    private ocUserGroupService: OcUserGroupService,
+    @Inject(applicationConfiguration) private appConfig: AppConfig
+  ) {}
+
+  ngOnInit() {
+    this.getUserGroup().subscribe((x) => (this.usergroup = x));
+  }
+
+  getUserGroup(): Observable<UserGroup> {
+    return this.activatedRoute.params.pipe(
+      flatMap((params) => {
+        if (params.userGroupID) {
+          return this.ocUserGroupService.Get(
+            this.appConfig.buyerID,
+            params.userGroupID
+          );
+        }
+      })
+    );
+  }
+
+  updateUserGroup(group: UserGroupUpdate): void {
+    if (!group.prevID) {
+      throw Error('Cannot update a user group without an ID');
+    }
+
+    this.ocUserGroupService
+      .Patch(this.appConfig.buyerID, group.prevID, group)
+      .subscribe((x) => (this.usergroup = x));
+  }
+}
+
+interface UserGroupUpdate extends UserGroup {
+  // ID can be changed in the form, so we need the previous ID to ensure we update the correct record
+  prevID: string;
+}
```

Comment:
Why don't you get the previous user group id from the route? Looks like its a route parameter.  You could save it to a private variable any time the parameter changes. That would let you get rid of this odd single-purpose interface, the validation check on `updateUserGroup`, and whatever sets `prevID` on the child. 

If you prefer to still get it from the child I'd prefer you emit it as a second value from `updateUserGroup` so you don't need a custom interface for it. Could emit:
`updateUserGroup(group: UserGroup, previousID: string)`